mne/preprocessing/nirs/_scalp_coupling_index.py

```diff
@@ -0,0 +1,62 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from ...io import BaseRaw
+from ...utils import _validate_type, verbose
+from ..nirs import _channel_frequencies, _check_channels_ordered
+from ...filter import filter_data
+
+
+@verbose
+def scalp_coupling_index(raw, l_freq=0.7, h_freq=1.5,
+                         l_trans_bandwidth=0.3, h_trans_bandwidth=0.3,
+                         verbose=False):
+    r"""Calculate scalp coupling index.
+
+    This function calculates the scalp coupling index [1]. This is a measure of
+    the quality of the connection between the optode and the scalp.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The raw data.
+    %(l_freq)s
+    %(h_freq)s
+    %(l_trans_bandwidth)s
+    %(h_trans_bandwidth)s
+    %(verbose)s
+
+    Returns
+    -------
+    sci : array of float
+        Array containing scalp coupling index for each channel.
+
+    References
+    ----------
+    .. [1] Pollonini, Luca, et al. "Auditory cortex activation to natural
```

Comment:
This is so painfully 2019. See how we do it in modern times:

https://github.com/mne-tools/mne-python/blob/master/mne/beamformer/_dics.py#L167-L171
https://github.com/mne-tools/mne-python/blob/master/doc/references.bib#L583-L592

which become

https://mne.tools/dev/generated/mne.beamformer.make_dics.html#hippetal2011

Reply:
So nerdy. Love it